doc/en/api/1.0.0/resource.yaml

```diff
@@ -0,0 +1,210 @@
+          required: false
+          default: default
+          enum:
+            - default
+            - metadata
+            - move
+            - copy
+        - name: format
+          in: query
+          type: string
+          description: The format of the response. Only applicable for the `metadata` operation, or for a directory resource.
+          required: false
+          default: html
+          enum:
+            - html
+            - xml
+            - json
+      responses:
+        200:
+          description: OK
+          headers:
+            Last-Modified: 
+              type: string
+              description: When resource was last modified.
+            Content-Type: 
+              type: string
+              description: Will guess mime-type from extension or content.
+            Resource-Type: 
+              type: string
+              description: Resource type (custom)
+              enum:
+                - directory
+                - resource
+            Resource-Parent: 
+              type: string
+              description: Path to parent (custom)
+        404:
+          description: Resource not found.
+    head:
+      operationId: resourceHead
+      description: Show resource metadata in HTTP headers.
+      responses:
+        200:
+          description: OK
+          headers:
+            Last-Modified: 
+              type: string
+              description: When resource was last modified.
+            Content-Type: 
+              type: string
+              description: Will guess mime-type from extension or content.
+            Resource-Type: 
+              type: string
+              description: Resource type (custom)
+              enum:
+                - directory
+                - resource
+            Resource-Parent: 
+              type: string
+              description: Path to parent (custom)
+        404:
+          description: Resource not found.
+    post: 
+      operationId: resourcePost
+      description: Invalid. Use PUT to create a resource.
+      responses:
+        405:
+          description: Method not allowed.
+    put:
+      operationId: resourcePut
+      description: Upload/move/copy a resource, create directories on the fly (overwrite if exists). For move/copy operations, place source path in body. Copying is not supported for directories.
+      parameters:
+        - name: operation
+          in: query
+          type: string
+          description: The type of PUT operation. `default` creates a new resource or alters an existing resource. `move` moves the resource to a new location. `copy` duplicates the resource to a new location
+          required: false
+          default: default
+          enum:
+            - default
+            - move
+            - copy
```

Comment:
It makes more sense for move and copy to be here, not sure why they were on `get`